### PR TITLE
more 7.108 items

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/README.md [hunk 1]
```diff
@@ -19,7 +19,14 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
 * NEW: Get verbose CLI version information in JSON format with just a single command. Use the new `--json` and `--verbose` parameters of `sfdx version`. It's now a smidgen easier to gather your environment details for us when you report an issue:
 
 	`sfdx version --json --verbose`
+	
+* NEW: Authorize an org using a Salesforce access token with the new `auth:accesstoken:store` command. Specify the instance URL where the org lives with the required `--instanceurl` parameter. When run interactively, the command prompts you for the access token. To use in a CI/CD script, set the new environment variable SFDX_ACCESS_TOKEN to the access token and run the command with the `--noprompt` parameter. 
+
 
+    ```bash
+    export SFDX_ACCESS_TOKEN=<access-token>
+    sfdx auth:accesstoken:store --instanceurl https://login.salesforce.com --noprompt
```
Comment (ekapner) [inline releasenotes/README.md]: Are there other required options like the org username? And it'll be interesting to see what's required for the instanceurl when those MyDomain changes kick in. Not sure if you want to remind yourself to update the command messages when this happens (if you're using this specific example in --help)

File: releasenotes/README.md [hunk 2]
```diff
@@ -19,7 +19,14 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
 * NEW: Get verbose CLI version information in JSON format with just a single command. Use the new `--json` and `--verbose` parameters of `sfdx version`. It's now a smidgen easier to gather your environment details for us when you report an issue:
 
 	`sfdx version --json --verbose`
```
Comment (ekapner) [inline releasenotes/README.md:21]: If you want --verbose output, you must also specify --json? Can you run these options separately? Or is now that you can get verbose JSON output? When I ran the command alone, with --json, and with --verbose, I got the exact same results/output for all 3 with current cli version.

**Review by ekapner — COMMENTED** [review]
A couple of questions - if you make updates, happy to review again.

File: releasenotes/README.md (hunk 2 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md:21]: Hmmm, i see very diff output with the various param combos.  Are you sure you're on the RC (7.108)?  I'll reword to make it clear you can use 'em separately.

Comment (ekapner) [reply releasenotes/README.md:21]: Just for kicks, I used the current CLI but wanted to see if these options did anything.

File: releasenotes/README.md (hunk 1 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: no, the access token sorta already includes the org and username. and i changed the example to use mydomain.

**Comment by jshackell-sfdc** [comment]
@ekapner , can you re-review?  thx!

File: releasenotes/README.md [hunk 3]
```diff
@@ -16,10 +16,18 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
-* NEW: Get verbose CLI version information in JSON format with just a single command. Use the new `--json` and `--verbose` parameters of `sfdx version`. It's now a smidgen easier to gather your environment details for us when you report an issue:
+* NEW: Get detailed CLI version information in JSON format with just a single command. Use the new `--json` parameter of `sfdx version` to get JSON output. Use the new `--verbose` parameter to get detailed information, such as the full list of installed plug-ins. Use them together to easily gather your environment details for us when you report an issue:
```
Comment (ekapner) [inline releasenotes/README.md:19]: Like the additional details. Nice.

**Review by ekapner — APPROVED** [review]

**Pull request merged**